app/views/interview_questions/edit_multiple.html.slim

```diff
@@ -0,0 +1,46 @@
+- provide(:title, 'Interview')
+.container.content-area#workflow
+  h1.center
+	  | Interview: 
+	  = @interview.id
+  br
+  .form
+    = form_tag update_multiple_interview_questions_path, method: :put do 
+      #tabs
+        ul
+          - num=0
+          - @interview.questions.each_with_index do |q|
+            - num+=1
+            - l="li"+num.to_s
+            - hrf="#tabs-"+num.to_s
+            li.d-line id=l
+              a href=hrf
+                = num 
+        br
+        - num_tab=0
+        - @interview.questions.each_with_index do |q|
+          - hrf="tabs-"+(num_tab+1).to_s
+          div id=hrf
+            h3
+              | Question 
+              = num_tab+1
+              | :  
+              = q.content
+            h4
+              | Answer: 
+              = q.answer
+            br
+            = fields_for "interview_questions[]", InterviewQuestion.where(interview_id: @interview.id, question_id: q.id).first do |f|
```

Comment:
@vvs79 if you don't understand my comment, please ask me via skype.
why didn't you use:
```ruby
@interview.interview_questions.each do |interview_question|
  ...
  = fields_for "interview_questions[]", interview_question
```
Your current change does not correlates with my comment and looks like an attempt to trick me =)